CONTRIBUTING.rst

```diff
@@ -5,6 +5,15 @@ Contributing
 Information About Contributing to This Manual
 =============================================
 
+Local rendering
+---------------
+
+You can render this file locally if you have `Docker <https://www.docker.com/>`
+and `make <https://www.gnu.org/software/make/>` installed on your local
+machine::
+
+    make docs
```

Comment:
And what is the alternative for WIndows users?

Reply:
WSL2.

Reply:
So we force the users to use Linux/Mac.

What about providing a `render.sh` and a `render.bat` file?

Reply:
We force no-one. they can use the Docker command as before?